spring-cloud-task-core/src/main/java/org/springframework/cloud/task/configuration/TaskProperties.java

```diff
@@ -62,6 +62,23 @@
 	 */
 	private Boolean closecontextEnabled = true;
 
+	/**
+	 * When set to true the
+	 * {@link org.springframework.cloud.task.listener.TaskLifecycleListener#doTaskStart()}
```

Comment:
We shouldn't be quite so specific on the implementation details in javadoc.  Say we wanted to change the implementation...how we cause this to happen is independent of the fact that it does.  Since javadoc is part of the contract, we need to be careful with how specific we are.